platform/admin/ui/src/test/js/unit/Export.js

```diff
@@ -0,0 +1,202 @@
+/*jslint node: true */
+/* global describe, it, require */
+'use strict';
+
+var requirejs = require('requirejs');
+requirejs.config({
+    baseUrl: '.',
+    nodeRequire: require,
+    packages: [
+        {
+            name: 'squirejs',
+            location: 'node_modules/squirejs',
+            main: 'src/Squire'
+        }
+    ]
+});
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var Squire = requirejs('squirejs');
+var document = require('jsdom').jsdom();
+var window = document.defaultView;
+require('jquery')(window);
+var $ = window.$;
+var dependencyInjector = new Squire();
+var dependencyInjectedRequire = dependencyInjector.mock('jquery', $);
```

Comment:
I like your idea of encapsulating the above if we can in some kind of common object or base prototype and putting that in the utility file.